eval/judge.py

```
@@ -0,0 +1,264 @@
+            return val
+    return None
+
+
+def judge_findings(
+    all_findings: dict[str, list[CanonicalFinding]],
+    corpus_dir: Path,
+) -> list[Verdict]:
+    gt_map = _load_all_ground_truth(corpus_dir)
+    all_verdicts: list[Verdict] = []
+
+    for _tool, findings in all_findings.items():
```

Ruff (0.15.12)

[warning] When using only the values of a dict use the `values()` method

Replace `.items()` with `.values()`

(PERF102)